src/algorithms/machinelearning/tensorflowpredict.h

```diff
@@ -53,6 +53,11 @@ class TensorflowPredict : public Algorithm {
   TF_SessionOptions* _sessionOptions;
   TF_Session* _session;
 
+  std::string _savedModel;
+  std::vector<std::string> _tags;
+  TF_Buffer* _runOptions;
+  TF_Buffer* _metaGraphDef;
```

Comment:
I can't see this variable being used anywhere in `tensorflowpredict.cpp`

Reply:
Addressed in #1186